api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -99,70 +92,56 @@
           FilterColumns.ETHNICITY.name(),
           FilterColumns.GENDER.name(),
           FilterColumns.RACE.name());
 
-  private CohortBuilderService cohortBuilderService;
-  private CohortReviewService cohortReviewService;
-  private BigQueryService bigQueryService;
-  private CohortQueryBuilder cohortQueryBuilder;
-  private ReviewQueryBuilder reviewQueryBuilder;
-  private UserRecentResourceService userRecentResourceService;
-  private Provider<DbUser> userProvider;
-  private ParticipantCohortStatusMapper participantCohortStatusMapper;
-  private CohortReviewMapper cohortReviewMapper;
-  private ParticipantCohortAnnotationMapper participantCohortAnnotationMapper;
+  private final CohortBuilderService cohortBuilderService;
+  private final CohortReviewService cohortReviewService;
+  private final BigQueryService bigQueryService;
+  private final CohortQueryBuilder cohortQueryBuilder;
+  private final ReviewQueryBuilder reviewQueryBuilder;
+  private final UserRecentResourceService userRecentResourceService;
+  private final Provider<DbUser> userProvider;
+  private final WorkspaceService workspaceService;
   private final Clock clock;
-  private static final Logger log = Logger.getLogger(CohortReviewController.class.getName());
 
   @Autowired
   CohortReviewController(
+      CohortReviewService cohortReviewService,
       BigQueryService bigQueryService,
       CohortBuilderService cohortBuilderService,
       CohortQueryBuilder cohortQueryBuilder,
-      CohortReviewService cohortReviewService,
       ReviewQueryBuilder reviewQueryBuilder,
       UserRecentResourceService userRecentResourceService,
       Provider<DbUser> userProvider,
-      ParticipantCohortStatusMapper participantCohortStatusMapper,
-      CohortReviewMapper cohortReviewMapper,
-      ParticipantCohortAnnotationMapper participantCohortAnnotationMapper,
+      WorkspaceService workspaceService,
       Clock clock) {
+    this.cohortReviewService = cohortReviewService;
     this.bigQueryService = bigQueryService;
     this.cohortBuilderService = cohortBuilderService;
     this.cohortQueryBuilder = cohortQueryBuilder;
-    this.cohortReviewService = cohortReviewService;
     this.reviewQueryBuilder = reviewQueryBuilder;
     this.userRecentResourceService = userRecentResourceService;
     this.userProvider = userProvider;
-    this.participantCohortStatusMapper = participantCohortStatusMapper;
-    this.cohortReviewMapper = cohortReviewMapper;
-    this.participantCohortAnnotationMapper = participantCohortAnnotationMapper;
+    this.workspaceService = workspaceService;
     this.clock = clock;
   }
 
   /**
    * Create a cohort review per the specified workspaceId, cohortId, cdrVersionId and size. If
    * participant cohort status data exists for a review or no cohort review exists for
    * cohortReviewId then throw a {@link BadRequestException}.
-   *
-   * @param workspaceNamespace
-   * @param workspaceId
-   * @param cohortId
-   * @param cdrVersionId
```

Comment:
Why is this removed?

Reply:
A few reasons, they didn't match the parameters being passed into the method and Intellij gives a warning for any `@param` that doesn't have a description attached to them.  I feel that they are self explanatory and don't need descriptions and was tired of seeing the warnings, so i removed them.